.github/workflows/pull-request.yaml

```diff
@@ -24,18 +24,18 @@ jobs:
         env:
           TYPE_TO_LABEL: |
             {
-              "feat":"kind/enhancement",
-              "fix":"fix",
-              "chore":"chore",
-              "docs":"kind/documentation",
-              "test":"kind/test",
-              "perf":"kind/performance"
+              "feat":"kind/feature",
+              "fix":"kind/bugfix",
+              "chore":"kind/chore",
+              "docs":"area/documentation",
```

Comment:
qq: why is it area documentation or area/testing instead of kind?

Reply:
we had both in the past: `area/` and `kind/` - since the labels are more used by PO on issues, than developers on PRs, I thought, I'll keep only the `area/`

Reply:
I think this might be something to revisit because any documentation we update in OCM (e.g. examples) would be labelled as such. Is this okay?

Reply:
> I think this might be something to revisit because any documentation we update in OCM (e.g. examples) would be labelled as such. Is this okay?

Why not? If we plan to get some assistance from an information-engineer to revisit all our documentation for example. But if you have concerns, we can remove that.

Reply:
I was expecting all PRs to have `kind` instead of `area` because 
- an `area` is where something belongs, while 
- `kind` is a definition of what something is

We are now unifying things into different prefixes and I really dont like it. But lets consider this as nit and not as blocking comment